src/command-helpers/build/validateDeploymentJson.js

```diff
@@ -262,10 +262,12 @@ function checkDuplicateIDs(deploymentJsonMap) {
   }
 }
 
-function checkProtocolArgumentCount(protocol, protocolData) {
-  if (Object.keys(protocolData).length !== 4) {
+function checkProtocolArguments(protocol, protocolData) {
+  const requiredKeys = ['schema', 'base', 'protocol', 'deployments']
```

Comment:
@tnkrxyz / @steegecs do you think we should add `project` to this list? Is it now a required key in the `deployment.json`?

Reply:
Yeah, I think we should require this.

Reply:
I didn't include 'project' because at least one subgraph didn't have a project tag, so I thought it is optional. Looks like #1781 fixed it; feel free to add 'project' to requiredKeys.